src/main/resources/sql/migration.0.sql

```diff
@@ -0,0 +1,114 @@
+CREATE TABLE IF NOT EXISTS bs_player
```

Comment:
This is not how we do migrations, because it allows only pure SQL

Reply:
This is a pure sql script that needs no other facilities to work, so it can be easily seperated into a readble script file.